docker-compose.yml

```diff
@@ -112,7 +112,7 @@ services:
      - "8985:8983"
     environment:
       - SOLR_OPTS=-XX:-UseLargePages
-      - SOLR_HEAP=800m
+      - SOLR_HEAP=3g
```

Comment:
Did we do any poking on this?   Is it really 3g?????    We probably need to call this out...

Reply:
I don't know what's a good limit. It is an artefact of the performance tuning and we might be able to reduce it again.